$OpenBSD: patch-acinclude_m4,v 1.1 2018/11/01 08:39:04 ajacoutot Exp $

/usr/bin/../lib/libcompiler_rt.a(gcc_personality_v0.o): In function `__gcc_personality_v0':                                              
/usr/src/lib/libcompiler_rt/gcc_personality_v0.c:205: undefined reference to `_Unwind_GetLanguageSpecificData'                           
/usr/src/lib/libcompiler_rt/gcc_personality_v0.c:209: undefined reference to `_Unwind_GetIP'                                             
/usr/src/lib/libcompiler_rt/gcc_personality_v0.c:210: undefined reference to `_Unwind_GetRegionStart'                                    
/usr/src/lib/libcompiler_rt/gcc_personality_v0.c:241: undefined reference to `_Unwind_SetGR'                                             
/usr/src/lib/libcompiler_rt/gcc_personality_v0.c:243: undefined reference to `_Unwind_SetGR'                                             
/usr/src/lib/libcompiler_rt/gcc_personality_v0.c:244: undefined reference to `_Unwind_SetIP'

augtool:/usr/lib/libreadline.so.4.0: undefined symbol 'tgetent'

Index: acinclude.m4
--- acinclude.m4.orig
+++ acinclude.m4
@@ -19,7 +19,7 @@ AC_DEFUN([AUGEAS_COMPILE_WARNINGS],[
 
     warnCFLAGS=
 
-    common_flags="-fexceptions -fasynchronous-unwind-tables"
+    common_flags=""
 
     case "$enable_compile_warnings" in
     no)
@@ -105,7 +105,7 @@ AC_DEFUN([AUGEAS_CHECK_READLINE], [
   AC_CHECK_HEADERS([readline/readline.h])
 
   # Check for readline.
-  AC_CHECK_LIB(readline, readline,
+  AC_CHECK_LIB(readline, tgetent,
           [use_readline=yes; READLINE_LIBS=-lreadline],
           [use_readline=no])
 
